.github/local-development.md

```diff
@@ -0,0 +1,107 @@
+# Local Development
+
+This document gives tips and tricks on how to create the Renovate documentation website locally to add features or fix bugs.
+You can improve this documentation by opening a pull request.
+For example, if you think anything is unclear, or you think something needs to be added, open a pull request!
+
+## Installation
+
+### Prerequisites
+
+You need the following dependencies for local development:
+
+- Git
+- Node.js `^12.13.0 || >=14.15.0`
+- Yarn `^1.17.0`
+- Python `^3.8`
+- Bash shell with `make`
+
+We support Node.js versions according to the [Node.js release schedule](https://github.com/nodejs/Release#release-schedule).
+
+_Linux_
+
+You can use the following commands on Ubuntu.
+
+```sh
+curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update
+sudo apt-get install -y git python-minimal build-essential nodejs yarn
+```
+
+_Windows_
+
+QUESTION: Git Bash for Windows does not come with `make` by default, how should a user work around this? Or is Windows not supported for making the docs locally?
```

Comment:
Do we even support Windows for the current build process?
The GitHub Actions use `ubuntu-latest`.

Reply:
Nope, i think windows users should use wsl.

Reply:
Okay will rewrite to point them to Windows Subsystem for Linux then.

Do we want them to use WSL1 or WSL2?